recipes/freeglut/all/conanfile.py

```diff
@@ -0,0 +1,124 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class freeglutConan(ConanFile):
+    name = "freeglut"
+    description = "Open-source alternative to the OpenGL Utility Toolkit (GLUT) library"
+    topics = ("conan", "freeglut", "opengl", "gl", "glut", "utility", "toolkit", "graphics")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/dcnieho/FreeGLUT"
+    license = "X11"
+    exports_sources = ["CMakeLists.txt", "*.patch"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "gles": [True, False],
+        "print_errors_at_runtime": [True, False],
+        "print_warnings_at_runtime": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "gles": False,
+        "print_errors_at_runtime": True,
+        "print_warnings_at_runtime": True,
+    }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("opengl/system")
+        self.requires('glu/system')
+        if self.settings.os == "Linux":
```

Comment:
Untested. Since you're the BSD specialist, you have the last word on this.
If accepted, this change should probably be applied to other places in this recipe.
```suggestion
        if self.settings.os in ("Linux", "FreeBSD"):
```